.github/workflows/rich-navigation.yml

```diff
@@ -0,0 +1,36 @@
+name: "Rich Navigation Indexing"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - release-*
+      - dbaeumer/richNav
```

Comment:
Should this be part of the production action?

Reply:
Sorry no. I added it to test it to make sure the script is OK. Since it is I will remove.